### PR TITLE
Added logrotate policy to rotate cccp.log. Fixes #236

### LINKED ISSUE #236: Add logrotate policies
## Issue

Currently, our log files are just growing and growing. We need to create log rotate policies to rotate logs and purge logs which are no longer in use

### PR DESCRIPTION
**Pull request by rtnpro**

### CODE REVIEW

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests

File: provisions/roles/jenkins/master/templates/cccp.logrotate.j2 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/srv/pipeline-logs/cccp.log {
```
Comment (mohammedzee1000) [inline provisions/roles/jenkins/master/templates/cccp.logrotate.j2]: There is nothing templaty here. Do we need a jinja template

**Comment by navidshaikh** [comment]
This PR was not reviewed.

**Comment by bamachrn** [comment]
did a review and had a discussion on this before merging

**Pull request merged**